Tasktive/Models/ViewModels/TasksViewModel.swift

```diff
@@ -167,6 +167,88 @@ final class TasksViewModel: ObservableObject {
         })
     }
 
+    func deleteTasks(by date: Date, indexSet: IndexSet) async -> Result<Void, UserErrors> {
+        let dateHash = getHashDate(from: date)
+        var mutableTask = tasks
+
+        indexSet.reversed().forEach { taskIndex in
+            let task = tasks[dateHash]!.at(taskIndex)!
+
+            let result = dataClient.delete(
+                by: task.id,
+                from: persistenceController.context,
+                of: CoreTask.self
+            )
+            .mapError {
+                let error: Error?
+                switch $0 {
+                case .notFound:
+                    logger.error("task not found")
+                    return UserErrors.deleteFailure
+                case let .crud(error: crudError):
+                    error = crudError as? CoreTagsk.CrudErrors
```

Comment:
Small typo here

Reply:
haha, my bad!

I tried to switch to terminal window in Mac and started typing "git" but the cursor was still in xCode 😆